lookout/style/format/rules.py

```diff
@@ -18,22 +18,45 @@
 from sklearn.tree import _tree as Tree, DecisionTreeClassifier
 
 from lookout.core.ports import Type
+from lookout.style.format.feature_extractor import FeatureGroup, IndexToFeature
+from lookout.style.format.features import Feature, get_features
+
+
+class RuleAttribute(NamedTuple(
+        "RuleAttribute", (("feature", int), ("cmp", bool), ("threshold", float)))):
+    """
+    Hold the feature, comparison operator and threshold of a split.
+
+    `feature` is the feature taken for comparison
+    `cmp` is the comparison type: True is "x > v", False is "x <= v"
+    `threshold` is "v", the threshold value
+    """
+
+    def describe(self, index_to_feature: IndexToFeature, features: Mapping[str, Feature]) -> str:
+        """Describe a rule attribute."""
```

Comment:
Format the rule attribute as text. We take features metadata to convert the integer indices to human-readable names.